README.md

```diff
@@ -47,7 +47,7 @@ Perl is distributed under a dual licence : [Artistic Licence](https://opensour
 - :man_with_turban: [Perl Monks](https://www.perlmonks.org/) - The Perl _Stack Overflow_
 - :man_with_turban: [Perl Monks… **again**](https://www.perlmonks.org/) - Because it's so cool, even if it looks like a cave :speak_no_evil:
 - :book: [Perldoc from perl.org](https://perldoc.perl.org/) and [perldoc from grinnz](https://perldoc.pl) :new:
-- :moneybag: [Core modules list](https://perldoc.perl.org/5.30.0/index-modules-A.html) - Web interface to the tool `corelist`
+- :moneybag: [Core modules list](https://metacpan.org/pod/index-modules-A) - Web interface to the tool `corelist`
```

Comment:
This is a meta-index page on perldoc.perl.org, the perldoc.pl equivalent would be https://perldoc.pl/modules.

Reply:
Ok I fixed it (https://metacpan.org/pod/index-modules-A is broken)